website/src/pages/ui.tsx

```diff
@@ -0,0 +1,115 @@
+          image={require("../../static/img/ui.png")}
+          bigImage={true}
+        />
+        <Hero
+          title="Cache stats"
+          subtitle="Debug cache misses and monitor cache performance. Examine individual invocations, or track cache hit ratios across builds."
+          image={require("../../static/img/blog/cache_misses.png")}
+          bigImage={true}
+          flipped={true}
+        />
+        <Hero
+          title="Trends"
+          subtitle="Track how your build performance is changing over time. Catch performance regressions and take credit for optimizations."
+          image={require("../../static/img/blog/trends.png")}
+          bigImage={true}
+        />
+        <Hero
+          title="Action explorer"
+          subtitle="View input files, command details, evironment variables, timing information, output files, and more for each individual action."
+          image={require("../../static/img/action-explorer.png")}
+          bigImage={true}
+          flipped={true}
+        />
+        <Hero
+          title="Invocation diffing"
+          subtitle={`Automatically compare two Bazel invocations to see what changed. Find out why the build "works on my machine".`}
+          image={require("../../static/img/blog/compare.png")}
+          bigImage={true}
+        />
+        <Hero
+          title="Dependency graph"
+          subtitle="Visualize the dependencies between your targets using Bazel's powerful query tool."
+          image={require("../../static/img/blog/query_graph.png")}
+          bigImage={true}
+          flipped={true}
+        />
+        <Hero
+          title="Team build history"
+          subtitle="View builds across your organization. Filter by date range, user, repo, branch, commit, host, CI, and more."
+          image={require("../../static/img/blog/date_picker.png")}
+          bigImage={true}
+        />
+        <Hero
+          title="Downloadable artifacts"
+          subtitle="Download the outputs of your builds from the remote cache with just a single click."
+          image={require("../../static/img/artifacts.png")}
+          bigImage={true}
+        />
+        <Hero
+          title="Test grid"
+          subtitle="Quickly find and fix flaky tests. Click on past failures to see test logs from failed runs to identify patterns."
+          image={require("../../static/img/test-grid.png")}
+          bigImage={true}
+          flipped={true}
+        />
+        <Hero
+          title="Suggestions"
+          subtitle="Get relevant suggestions on how to improve your builds surfaced inline — right when you need them."
+          image={require("../../static/img/blog/suggested-fixes.png")}
+          bigImage={true}
+        />
+        <Hero
+          title="Live updating"
+          subtitle="The real-time invocation page that shows you the current state of your build, whether it's local or on CI."
```

Comment:
nit: "See the current state of your build in real-time, whether it's local or on CI."

Reply:
Fixed